docs/usage/general.rst

```diff
@@ -45,6 +45,12 @@ that is the safer way; see the warning below.
     experience errors and read broken data if changes to that repository are
     being made at the same time.
 
+Option ``--help`` when used as a command for help on subcommands(eg. ``borg help compact``), 
+works as expected. But, it does not work when used on sub-sub-commands(eg. ``borg help key export``).
+There is a workaround to get help on sub-sub-commands using the help command as a flag(eg. 
+``borg key export --help``).
```

Comment:
Please proofread this again.

Reply:
Done. Can you please confirm and close this issue?